packages/neon/neon/lib/src/utils/push_utils.dart

```diff
@@ -76,15 +76,31 @@ class PushUtils {
         final cache = Cache(platform);
         await cache.init();
 
-        NextcloudNotificationsNotification? notification;
         var accounts = <Account>[];
         Account? account;
+        NextcloudNotificationsNotification? notification;
+        AndroidBitmap<Object>? largeIconBitmap;
         try {
           accounts = loadAccounts(AppStorage('accounts', sharedPreferences));
           account = accounts.find(instance);
           if (account != null) {
             notification =
                 (await account.client.notifications.getNotification(id: pushNotification.subject.nid!)).ocs.data;
+            if (notification.icon != null) {
+              final cacheManager = DefaultCacheManager();
+              final file = await cacheManager.getSingleFile(notification.icon!);
+              const iconSize = 256;
+              late Bitmap bitmap;
+              if (notification.icon!.endsWith('.svg')) {
+                final pictureInfo = await vg.loadPicture(SvgStringLoader(file.readAsStringSync()), null);
+                final image = await pictureInfo.picture.toImage(iconSize, iconSize);
+                final bytes = await image.toByteData();
+                bitmap = Bitmap.fromHeadless(iconSize, iconSize, bytes!.buffer.asUint8List());
+              } else {
+                bitmap = Bitmap.fromHeadless(iconSize, iconSize, file.readAsBytesSync());
+              }
+              largeIconBitmap = ByteArrayAndroidBitmap(bitmap.content);
```

Comment:
We are assigning `bytes!.buffer.asUint8List()` and `file.readAsBytesSync()` to content and then access it again a line below without any changes to it?

```dart
class Bitmap {
  Bitmap.fromHeadless(this.width, this.height, this.content);

 [...]

  /// The width in pixels of the image.
  final int width;

  /// The width in pixels of the image.
  final int height;

  /// A [Uint8List] of bytes in a RGBA format.
  final Uint8List content;
  
  [...]
}
```

Unless we use one of the other Bitmap constructors or actually need a bitmap transformation (that image might be able to do) we don't need to depend on this package at all.

Reply:
As far as I know you need an actual bitmap image

Reply:
But unless I'm missing something we are not making use of the `Bitmap` class.

Reply:
You were right, I wasn't paying any attention...